cli/src/commands/node/mod.rs

```diff
@@ -193,6 +194,9 @@ impl Node {
                 timeouts: P2pTimeouts::default(),
                 chain_id: CHAIN_ID.to_owned(),
                 peer_discovery: !self.no_peers_discovery,
+                initial_time: SystemTime::now()
```

Comment:
Why do we have duplicate state here? Maybe it can be avoided? Also use `redux::Timestamp` everywhere for time, never `SystemTime`.

Reply:
If it's used just for pubsub nonce, then please change the name.